### ISSUE TITLE
Running labman config to create config file fails because ... config file doesn't already exist?

### ISSUE THREAD

**Issue by AmandaBirmingham** [jdereus/LabControl#450]
Tried to run `labman config` to create a config file for an environment that does not currently have one (either in `~/.labman.cfg` or in some other location specified by `environ['LABMAN_CONFIG_FP']`) and got this error:

```
(labman) Amandas-MacBook-Pro-6:labman Birmingham$ labman config
Traceback (most recent call last):
  File "/Users/Birmingham/Work/Repositories/Labman/labman/labman/db/configuration_manager.py", line 109, in __init__
    self.conf_fp = environ['LABMAN_CONFIG_FP']
  File "/Users/Birmingham/Applications/miniconda3/envs/labman/lib/python3.6/os.py", line 669, in __getitem__
    raise KeyError(key) from None
KeyError: 'LABMAN_CONFIG_FP'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/Birmingham/Applications/miniconda3/envs/labman/bin/labman", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/Users/Birmingham/Work/Repositories/Labman/labman/scripts/labman", line 15, in <module>
    from labman.db.environment import patch_database
  File "/Users/Birmingham/Work/Repositories/Labman/labman/labman/db/environment.py", line 14, in <module>
    from labman.db import sql_connection
  File "/Users/Birmingham/Work/Repositories/Labman/labman/labman/db/sql_connection.py", line 23, in <module>
    from . import settings
  File "/Users/Birmingham/Work/Repositories/Labman/labman/labman/db/settings.py", line 12, in <module>
    labman_settings = configuration_manager.ConfigurationManager()
  File "/Users/Birmingham/Work/Repositories/Labman/labman/labman/db/configuration_manager.py", line 114, in __init__
    'Please, configure labman using `labman config`. If the '
RuntimeError: Please, configure labman using `labman config`. If the config file is not in `~/.labman.cfg`, please set the `LABMAN_CONFIG_FP` environment variable to the configuration file
```
So ... I can't make a config file because I don't already have a config file?  I see that for travis this is not a problem as the .travis.yml copies the test config to ~/.labman.cfg:

https://github.com/jdereus/labman/blob/0b05cd0da7a8acaecfc2092ed34d0dbe2229fb6b/.travis.yml#L55-L56

... but won't new users need to use `labman config` to create a config from nothing?

**Comment by charles-cowart** [comment]
@AmandaBirmingham is it a blocker for you? We should resolve this before full launch in either case; doesn't seem like it should be too difficult.

**Comment by AmandaBirmingham** [comment]
@charles-cowart  Nope, not a blocker: the work-around is to manually do what the `.travis.yml` code does and copy the `labman/db/support_files/test_config.cfg` file to `~/.labman.cfg`.  Once it exists (even empty) then `labman config` runs like a dream.  So the quick and dirty fix would be to add a step to the install instructions saying a human needs to copy this file before running `labman config`.  If we ever get to the point of installing labman with `pip` or whatever instead of just cloning it from github, we could make the installer do the copy.

Still, seems suboptimal that the code demands that file we want to create already exist :)

**Comment by charles-cowart** [comment]
Resolved with #554. The support_files directory was not being installed into the environment.
